### PR TITLE
fix(transactions): send user registration status to message service

### PR DESCRIPTION
**Pull request by armandopmj**

### CODE REVIEW

**Comment by mLuby** [comment]
@armandopmj can I merge this?

File: server/api/transactions/transactions.controller.js [hunk 1]
```diff
@@ -103,6 +103,7 @@ function chargeUser (data, callback) {
 }
 function updateTransaction (data, callback) {
   if( data.transaction ) {
+    console.log( 'inside updateTransaction, data:', data );
```
Comment (mLuby) [inline server/api/transactions/transactions.controller.js]: @armandopmj left a console.log here

File: server/api/users/users.services.js [hunk 2]
```diff
@@ -1,12 +1,31 @@
...
-exports.constructSignUpGiftOptionsMessages = gifts => {
-  const message = `Hey! You have successfully signed up for SurpriseHer monthly limited gift lists! Here is our curated gift selection only avalible to new signups:\n\n
-  ${constructGiftOptionsStr(gifts)} PRODUCT URL\n\n
-  to begin the order process, txt back the gift code and the day in month you'd like us to ship (e.g 3 ${gifts[0].look_up}${gifts[0].gift_id}, 18 ${gifts[1].look_up}${gifts[1].gift_id})`
+const constructSignUpGiftOptionsMessages = (userRegistered, gifts, addresses) => {
+  let message = ''
+  if( userRegistered ) {
+    const addressCodes = addresses.map( address => address.code_name)
+    message = `Hey! You have successfully signed up for SurpriseHer monthly limited gift lists! Here is our curated gift selection only avalible to new signups:
+    \n\n${constructGiftOptionsStr(gifts)}PRODUCT URL
```
Comment (mLuby) [inline server/api/users/users.services.js]: @armandopmj es6 backticks respect newlines, so unless you actually want 3 newlines, this might be too many.

**Comment by mLuby** [comment]
@armandopmj made a few comments. Let me know when you think it's ready to merge.

File: server/api/users/users.services.js (hunk 2 above)
Comment (armandopmj) [inline server/api/users/users.services.js]: @mLuby ah tricky thanks!

**Pull request merged**